storybook/src/stories/Form/FilterDropdown.stories.tsx

```diff
@@ -0,0 +1,239 @@
+import React, { useState, useCallback } from 'react';
+import { IconSVGs } from '@future-standard/icons';
+import styled from 'styled-components';
+import { boolean, text, select, number } from "@storybook/addon-knobs";
+import { action } from '@storybook/addon-actions';
+import {
+  FilterDropdown,
+  IFilterDropdownValue,
+  PageHeader,
+} from 'scorer-ui-kit';
+
+export default {
+  title: 'Form/atoms',
```

Comment:
Can we move filter specific parts to the `Filters` section as they'll always have that association but aren't really an input form in the same way as other things here.